### DISCUSSION TITLE
Is there music support?

### DISCUSSION THREAD

**Discussion by disclearing** [JulianPrieber/littlelink-custom#614] [Questions]
Is there music support?

**Comment by JulianPrieber** [comment]
There is no music support outside sites like SoundCloud etc.

**Reply by disclearing** [reply]
no like could the user upload a .mp4 as a background etc ? and allow it to play music I have not had the chance to look through the src code yet

**Reply by JulianPrieber** [reply]
We don't have a system like that in the app.

**Reply by disclearing** [reply]
will there ever be plans for it

**Reply by JulianPrieber** [reply]
In the form of autoplaying music in the background, definitely not.
We might implement a SoundCloud music block.

You should be able to easily do this yourself with a custom theme.
